README.md

```diff
@@ -1,2 +1,45 @@
 # parse-hamnosys
 Sign language HamNoSys notation parsing tool.
+
+## Usage
+```
+$ python3 parse-hamnosys.py -sf <source_file> -df <destination_file>
+```
+
+## Usage example
+```
+$ python3 parse-hamnosys.py -sf hamnosys_example.txt -df hamnosys_parsed.txt
+```
+
+## Source file format
+Default input file format is defined by the [HearAI](https://github.com/hearai/hearai) project requrements. As in[hamnosys_example.txt](hamnosys_example.txt) parser requires a file that has 6 columns separated with a space sign " ". Due to this, if any of the columns contains space, it must be removed or replaced (for example with "_" sign) before passing to the parser. Parsers operates only on HamNoSys notation, that is stored in the last (6th) column. It shall start with HamNoSys sign (No quote nor apostrophe sign is allowed).
+Input file columns and description:
+* Name - name of a video file that given notation refers to
+* Start - sign start time (on a video)
+* End - sign end time (on a vide)
+* Dict - word number in a dictionary
+* Word - translation to a spoken language
+* Hamnosys - Notation 
+
+## Destination File format
+Destination file consists of following columns separated by the space " " sign:
+* Name - name of a video file that given notation refers to, directly copied from source file
+* Start - sign start time (on a video), directly copied from source file
+* End - sign end time (on a vide), directly copied from source file
+* Symmetry operator - Number that represents one of the classes (please refer to json file), parsed from notation
+* Dominant - Handshape - Baseform - Number that represents one of the classes (please refer to json file), parsed from notation
+* Dominant - Handshape - Thumb position - Number that represents one of the classes (please refer to json file), parsed from notation
+* Dominant - Handshape - bending - Number that represents one of the classes (please refer to json file), parsed from notation
+* Dominant - Handposition - extended finger direction - Number that represents one of the classes (please refer to json file), parsed from notation
+* Dominant - Handposition - palm orientation - Number that represents one of the classes (please refer to json file), parsed from notation
+* Dominant - Handposition - LR - Number that represents one of the classes (please refer to json file), parsed from notation
+* Dominant - Handposition - TB - Number that represents one of the classes (please refer to json file), parsed from notation
+* Dominant - Handposition - Distance - Number that represents one of the classes (please refer to json file), parsed from notation
+
```

Comment:
I think now you have more types of features in python file - with nondominant hand, doesn't it?
Please add somwere the number of classes.

Reply:
For now non dominan hand is not ready, i will update readme together with the code.
Font info and classes (number and mapping) added to the readme